### PR TITLE
Remove suddenDeath option from incremental feeds

### PR DESCRIPTION
**Pull request by talzich**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11011

## Description
The parameter "feedExpirationPolicy" offers an option called "suddenDeath." However, this option is only available for fully fetched integrations, not incremental feeds.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]
great!!!

**Comment by talzich** [comment]
Won't update `demisto/elasticsearch` docker image in `FeedElasticSearch.yml` because of breaking changes. The issue is handled by [this ticket](https://jira-dc.paloaltonetworks.com/browse/CIAC-10581) and @ShacharKidor will update the feed integration once her PR is merged.

File: Packs/FeedCyCognito/ReleaseNotes/1_0_22.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CyCognito Feed
+
+- Deprecated the suddenDeath option
```
Comment (JudahSchwartz) [inline Packs/FeedCyCognito/ReleaseNotes/1_0_22.md:6]:
```suggestion
- Removed the *suddenDeath* option from the *feedExpirationPolicy* parameter.
```
same with the other ones

Comment (talzich) [reply Packs/FeedCyCognito/ReleaseNotes/1_0_22.md]: done

**Pull request merged**